index.bs

```diff
@@ -23,6 +23,10 @@ spec: HTML; urlPrefix: https://html.spec.whatwg.org/multipage/
     type: dfn
         urlPrefix: media.html
             text: media element event task source
+    type: dfn
+        urlPrefix: interaction.html
+            text: traversable navigable
```

Comment:
This <dfn> is exported by the HTML, so I don't think you need to add it here explicitly.

Reply:
I had to because of:

```
LINK ERROR: No 'dfn' refs found for 'system visibility state' that are marked for export.
<a data-link-type="dfn" data-lt="system visibility state">system visibility state</a>
LINK ERROR: Multiple possible 'traversable navigable' dfn refs.
Arbitrarily chose https://html.spec.whatwg.org/multipage/document-sequences.html#nav-traversable
To auto-select one of the following refs, insert one of these lines into a <pre class=link-defaults> block:
spec:html; type:dfn; for:navigable; text:traversable navigable
spec:html; type:dfn; for:/; text:traversable navigable
<a data-link-type="dfn" data-lt="traversable navigable">traversable navigable</a>
LINK ERROR: No 'dfn' refs found for 'context object' that are marked for export.
```

Reply:
"system visibility state" isn't exported, but "traversable navigable" is. The error you're getting for the latter can probably be solved by using `<a for=/>traversable navigable</a>` like https://github.com/whatwg/storage/blob/main/storage.bs#L259 does.

Reply:
Fixed. Thanks!